backend/OCCUR/urls.py

```diff
@@ -20,15 +20,14 @@
 
 urlpatterns = [
     path('',views.apiUrlsList, name="apiUrlsList" ),
-    path('api/v1/new/resource/', views.ResourceCreate.as_view()),
-    path('api/v1/<int:id>/location', views.LocationRetrieveUpdateDestroy.as_view()),
-    path('api/v1/<int:id>/resource', views.ResourceRetrieveUpdateDestroy.as_view()),
-    path('api/v1/list/resource', views.ResourceList.as_view()),
-    path('api/v1/list/location', views.LocationList.as_view()),
-    path('api/v1/list/admin', admin_views.get_admins),
-    path('api/v1/<id>/delete/admin', admin_views.delete_admin),
-    path('api/v1/new/admin', admin_views.new_admin),
-    path('api/v1/<id>/update/admin', admin_views.update_admin),
-    path('admin/', admin.site.urls
-    ),
+    path('resources', views.ResourceListCreate.as_view()),
```

Comment:
These look way cleaner. We need to make sure to notify everyone when this is pushed tho